doc/source/cluster/commands.rst

```diff
@@ -136,30 +169,44 @@ To download or upload files to the cluster head node, use ``ray rsync_down`` or
 
 .. _monitor-cluster:
 
-Monitoring cluster status (``ray dashboard/monitor``)
+Monitoring cluster status (``ray dashboard/status``)
 -----------------------------------------------------
 
-The ray also comes with an online dashboard. The dashboard is accessible via HTTP on the head node (by default it listens on ``localhost:8265``). You can also use the built-in ``ray dashboard`` to do this automatically.
+The Ray also comes with an online dashboard. The dashboard is accessible via
+HTTP on the head node (by default it listens on ``localhost:8265``). You can
+also use the built-in ``ray dashboard`` to do this automatically.
 
 .. code-block:: shell
 
     $ ray dashboard cluster.yaml
 
-You can monitor cluster usage and auto-scaling status by tailing the autoscaling
-logs in ``/tmp/ray/session_*/logs/monitor*``.
+You can monitor cluster usage and auto-scaling status by running (on the head node)
 
 .. code-block:: shell
 
-    $ ray monitor cluster.yaml
+    $ ray status
```

Comment:
nice, we definitely need to implement `ray status yaml`

Reply:
+1, do we have that feature request tracked somewhere?